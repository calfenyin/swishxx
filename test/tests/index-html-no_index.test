index | | -e html:*.html -i html.index -C no_index -v1 | html | 0
