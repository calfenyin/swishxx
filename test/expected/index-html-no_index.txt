
index: done:
  6 indexed
  2482 words, 1310 indexed, 528 unique

